requirements/env_climada.yml

```diff
@@ -5,6 +5,7 @@ dependencies:
   - blas=1.0
   - bottleneck=1.3.2
   - cartopy=0.17.0
+  - conda-forge::cfgrib=0.9.7.7
```

Comment:
just to keep in line with the other dependencies I'd move it to the `-pip` section and list `- cfgrib==0.9.7.7`
unless there is an issue with the pip installation

Reply:
If I have understood the installation guide correctly (https://pypi.org/project/cfgrib/ : "alternatively, if you install the binary dependencies yourself, you can install the Python package from PyPI") The installation of cfgrib via pip only works if every user installs the binary dependencies (eccodes) themselves. Is this the prefered option?